### ISSUE TITLE
pyatv implicitly logged out, logging in again

### ISSUE THREAD

**Issue by niclas0219** [postlund/pyatv#1000]
By a coincidence I was looking through my logs of home-assistant and saw that the same message was spammed over and over again. I will start by removing the remote from apple tv and add again. I have the second or third gen apple tv.

The message is repeated about 12 times every second which doesn't seem like a good behaviour. Some kind of delay should be introduced somewhere?

```log
2021-04-03 13:00:15 INFO (MainThread) [pyatv.dmap.daap] Logged in and got session id 232771202
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.dmap] Playstatus error occurred: failed to login: 406
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.dmap] Waiting for playstatus updates
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.dmap.daap] Already logged in, re-using seasion id 232771202
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.support.net] GET URL: http://192.168.100.28:3689/ctrl-int/1/playstatusupdate?session-id=232771202&revision-number=214
2021-04-03 13:00:15 WARNING (MainThread) [homeassistant.components.apple_tv.media_player] A <class 'pyatv.exceptions.AuthenticationError'> error occurred: failed to login: 406
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.support.net] Response: status=403, headers=[Date=Sat, 03 Apr 2021 10:48:50 GMT, DAAP-Server=iTunes/11.1b37 (OS X), Content-Type=application/x-dmap-tagged, Content-Length=0]
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.support.net] << GET (Data=)
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.dmap.daap]
2021-04-03 13:00:15 INFO (MainThread) [pyatv.dmap.daap] implicitly logged out, logging in again
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.support.net] GET URL: http://192.168.100.28:3689/login?pairing-guid=0x4A5861992AC66A69&hasFP=1
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.support.net] Response: status=200, headers=[Date=Sat, 03 Apr 2021 10:48:50 GMT, DAAP-Server=iTunes/11.1b37 (OS X), Content-Type=application/x-dmap-tagged, Content-Length=47, Content-Encoding=gzip]
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.support.net] << GET (Data=6d6c6f67000000186d73747400000004000000c86d6c69640000000472aa6ea3)
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.dmap.daap] mlog: [container, dmap.loginresponse]
  mstt: 200 [uint, dmap.status]
  mlid: 1923772067 [uint, dmap.sessionid]

2021-04-03 13:00:15 INFO (MainThread) [pyatv.dmap.daap] Logged in and got session id 1923772067
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.support.net] GET URL: http://192.168.100.28:3689/ctrl-int/1/playstatusupdate?session-id=1923772067&revision-number=214
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.support.net] Response: status=403, headers=[Date=Sat, 03 Apr 2021 10:48:50 GMT, DAAP-Server=iTunes/11.1b37 (OS X), Content-Type=application/x-dmap-tagged, Content-Length=0]
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.support.net] << GET (Data=)
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.dmap.daap]
2021-04-03 13:00:15 INFO (MainThread) [pyatv.dmap.daap] implicitly logged out, logging in again
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.support.net] GET URL: http://192.168.100.28:3689/login?pairing-guid=0x4A5861992AC66A69&hasFP=1
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.support.net] Response: status=200, headers=[Date=Sat, 03 Apr 2021 10:48:50 GMT, DAAP-Server=iTunes/11.1b37 (OS X), Content-Type=application/x-dmap-tagged, Content-Length=47, Content-Encoding=gzip]
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.support.net] << GET (Data=6d6c6f67000000186d73747400000004000000c86d6c6964000000046ce97f80)
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.dmap.daap] mlog: [container, dmap.loginresponse]
  mstt: 200 [uint, dmap.status]
  mlid: 1827241856 [uint, dmap.sessionid]

2021-04-03 13:00:15 INFO (MainThread) [pyatv.dmap.daap] Logged in and got session id 1827241856
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.dmap] Playstatus error occurred: failed to login: 403
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.dmap] Waiting for playstatus updates
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.dmap.daap] Already logged in, re-using seasion id 1827241856
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.support.net] GET URL: http://192.168.100.28:3689/ctrl-int/1/playstatusupdate?session-id=1827241856&revision-number=214
2021-04-03 13:00:15 WARNING (MainThread) [homeassistant.components.apple_tv.media_player] A <class 'pyatv.exceptions.AuthenticationError'> error occurred: failed to login: 403
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.support.net] Response: status=403, headers=[Date=Sat, 03 Apr 2021 10:48:50 GMT, DAAP-Server=iTunes/11.1b37 (OS X), Content-Type=application/x-dmap-tagged, Content-Length=0]
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.support.net] << GET (Data=)
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.dmap.daap]
2021-04-03 13:00:15 INFO (MainThread) [pyatv.dmap.daap] implicitly logged out, logging in again
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.support.net] GET URL: http://192.168.100.28:3689/login?pairing-guid=0x4A5861992AC66A69&hasFP=1
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.support.net] Response: status=200, headers=[Date=Sat, 03 Apr 2021 10:48:50 GMT, DAAP-Server=iTunes/11.1b37 (OS X), Content-Type=application/x-dmap-tagged, Content-Length=47, Content-Encoding=gzip]
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.support.net] << GET (Data=6d6c6f67000000186d73747400000004000000c86d6c69640000000407f4b112)
2021-04-03 13:00:15 DEBUG (MainThread) [pyatv.dmap.daap] mlog: [container, dmap.loginresponse]
  mstt: 200 [uint, dmap.status]
  mlid: 133476626 [uint, dmap.sessionid]
```
**To Reproduce**
Don't know

**Expected behavior**

**System Setup (please complete the following information):**
 - OS: Centos 8 using docker-ce with latest home-assistant/stable
 - Python: home-assistant/stable
 - pyatv: [e.g. 0.3.10]
- Apple TV: gen 2/3

**Additional context**

**Comment by postlund** [comment]
Yeah, this definitely looks like a bug to me. The `revision-number` needs to be reset/re-fetched between logins at it'a likely outdated. Seems strange this hasn't been caught earlier 🤔 Thanks for reporting, I will look into it.

**Comment by postlund** [comment]
Yey on being number 1000 btw! 😎

**Comment by postlund** [comment]
This should now be fixed by #1004. It's too late to get it into 2021.4.1, so I'll delay this a bit and either get it into a patch release or the next regular release.
